docs/ThirdPartyHosting.md

```diff
@@ -69,7 +73,19 @@ Specifically, third-party hosts should not enable the following policies without
 
 Third-party users who are granted access to an internal lightning node or hot wallet functionality to enable features such as Payjoin, should understand the risk and trust associated with [using hot wallets](HotWallet.md) before choosing to use it. Use one of the [recommended wallets](WalletSetup.md) which provide an xpubkey to use in your store, if you are unsure which wallet type to use. 
 
-## Summary
-Third-party hosts play an important role in the ecosystem since they provide an easy and cost-effective way for users to try and use BTCPay Server. The role of honest hosts who provide free service to others is essential in the early phase of BTCPay Server adoption. However, users should be familiar with the pros, cons and potential risks involved when using a trusted third-party. Find the optimal balance between your use-case, cost, and privacy/security trade-offs.
+## Can I enable the use of my Lightning Network node to others ?
```

Comment:
imo this is better suited as a part of FAQ of some sort, this is not of huge significance to be included in an important readme like this one, but can be an external link to some other place.

Reply:
Oh which part are you talking about ? The white original text, the red summary bumped to the top, or the green title I added ?

Reply:
Exactly where I left my comment the entire  `Can I enable the use of my Lightning Network node to others ?` section should be in FAQ as it's a question.

Reply:
Oh ! 
I feel like it's to big of an answer for the FAQ (image + warnings) as the FAQ should be short question, short answer ?
I can change the question to be affirmative, and include the exacte same question in the FAQ leading back to this §.

Reply:
> the FAQ should be short question, short answer ?

No. Not really ,FAQ are frequent questions, regardless of their length. You can link to it from the ` Are there any limitations in features when using a third party host?` section.

Reply:
Okay, so I propose to delete that section from this current file, and add it in the [server settings faq](https://docs.btcpayserver.org/FAQ/FAQ-ServerSettings/#policies), under "policies".

ACK ?

Reply:
I'd rather have that one in Lightning-FAQ, imo people who'd want to use that are more likely to search it there, or at least cross-reference it somehow.